cEpiabm/src/dataclasses/person.hpp

```diff
@@ -41,18 +34,20 @@ namespace epiabm
         size_t m_mcellPos; // Position of person within Microcell::m_people;
         size_t m_household = 0; // Household's index within Microcell::m_household;
         size_t m_microcell; // Microcell's index within Cell::m_microcells;
-        bool m_hasHousehold = false; // flag for whether the household has been set;
+        bool m_hasHousehold; // flag for whether the household has been set;
+
+        std::set<size_t> m_places; // Indices of Places which the person is a member of;
 
     public:
         Person(size_t microcell, size_t cellPos, size_t mcellPos);
-        ~Person() = default;
-        Person(const Person&) = default;
-        Person(Person&&) = default;
+        ~Person();
+        //Person(const Person&) = default;
+        //Person(Person&&) = default;
```

Comment:
Should these still be commented?

Reply:
Commented because they're unused and are technically source implementations in the header file. Not sure what the workaround is, but these lines were causing whole repository to be recompiled even if unrelated files were changed.